app/views/recipes/public.html.erb

```diff
@@ -0,0 +1,12 @@
+<aside class="col-md-6 mx-auto">
+  <h1 class="text-center mb-4">Public Recipes</h1>
+  <% @recipes.each do |recipe| %>
+    <div class="card mb-3">
+      <div class="card-body">
+        <h5 class="card-title"><%= recipe.name %></h5>
+        <p class="card-text">By: <%= recipe.user.name %></p>
+        <p class="mb-0">Description: <%= recipe.description %></p>
+      </div>
+    </div>
+  <% end %>
+</aside>
```

Comment:
- [ ] Nice work following the given **wireframe** for the public recipe list :+1: However, it would be nice to make each public recipe clickable. so when clicking on a recipe item it lead to the recipe details.